drivers/dide/R/batch.R

```diff
@@ -19,6 +19,8 @@ template_data_task_run <- function(task_id, config, path_root) {
     remote_path(path_to_task_file(path_root, task_id, "Renviron"),
                 config$shares, config$platform)
 
+  data$redis_url <- config$redis_url %||% "10.0.2.254"
```

Comment:
That hardcoded IP ought to be an API output - it does come out of
https://mrcdata.dide.ic.ac.uk/hpc/api/v1/get_clusters/

{"clusters": [{
    "name": "wpia-hn.hpc",
    "buildqueue": "BuildQueue",
    "alias": "sk",
    "maxcores": "32",
    "redis": "10.0.2.254"
}]}


but that name (wpia-hn.hpc) is weird - this might be something half-baked/legacy/that we never used. Future ticket...

(Commented here about 10.0.2.254 being hard-coded in the task run scripts - then saw you'd fixed that :-) )

Reply:
yeah, it's a bit weird though because it's the internally facing ip for the server - so I'm mostly sidestepping for now. Let's see how the non-infiniband-linux-version goes and we can see if it's worth keeping the complexity there